ascii-logo.txt

```diff
@@ -1,37 +1,37 @@
-              ::::      ::::      ::::       ::::      ::::       ::::                        
```

Comment:
Can you make this a pure file rename? Since the file changes just white space related

Reply:
Done.